### PR TITLE
Fixed little error in doc, in the building process

### PR DESCRIPTION
**Pull request by gabrielcuvillier**
Just recompiled fastcomp while reading the doc, and noticed this minor issue (missing 'src'). Unless I misunderstood something.

### CODE REVIEW

**Comment by gabrielcuvillier** [comment]
well, on my machine I have the following folder structure:

emsdk
    clang
       fastcomp
          build_incoming_64
          src

So there is this "src" folder, from which cmake is able to compile stuff. But I don't know, maybe I did something wrong. When I run cmake command from "fastcomp", nothing happens as there is no CMakeList file. 

I have used the doc article "How to build from sources using the SDK"
http://kripken.github.io/emscripten-site/docs/building_from_source/building_emscripten_from_source_using_the_sdk.html

**Comment by kripken** [comment]
Oh, I guess `src/` might be something the emsdk made, to compile it from source (as opposed to downloading a binary build). So we need this in one of those two places, I'll comment.

File: site/source/docs/building_from_source/building_emscripten_from_source_using_the_sdk.rst [hunk 1]
```diff
@@ -116,7 +116,7 @@ After installing the main repositories:
 
 		::
 
-			cmake .. -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="host;JSBackend" -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF -DCLANG_INCLUDE_TESTS=OFF
+			cmake ../src -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="host;JSBackend" -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF -DCLANG_INCLUDE_TESTS=OFF
```
Comment (kripken) [inline site/source/docs/building_from_source/building_emscripten_from_source_using_the_sdk.rst]: @juj is this necessary for the emsdk, which creates a `src/` directory?

File: site/source/docs/building_from_source/building_fastcomp_manually_from_source.rst [hunk 2]
```diff
@@ -64,7 +64,7 @@ To build the Fastcomp code from source:
 
   ::
 
-    cmake .. -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="host;JSBackend" -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF -DCLANG_INCLUDE_TESTS=OFF
+    cmake ../src -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="host;JSBackend" -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF -DCLANG_INCLUDE_TESTS=OFF
```
Comment (kripken) [inline site/source/docs/building_from_source/building_fastcomp_manually_from_source.rst]: this change is unneeded I think - when not using the emsdk, there is no `src/` dir.

File: site/source/docs/building_from_source/building_emscripten_from_source_using_the_sdk.rst (hunk 1 above)
Comment (juj) [reply site/source/docs/building_from_source/building_emscripten_from_source_using_the_sdk.rst]: Oh hmm - this is quite awkward. This documentation is under section `Building Emscripten using your own fork`, which documents the set of steps that one would use to replace each of Emscripten, Emscripten-fastcomp and Emscripten-fastcomp-clang with one's own remote forks, and rebuild each. What makes this awkward is that this feels like something people would very rarely want to do: instead what practically everyone would want is to use one's own fork of Emscripten, but reuse kripken/emscripten-fastcomp and kripken/emscripten-fastcomp-clang unchanged.

If one is using emsdk to bootstrap Emscripten+emscripten-fastcomp+emscripten-fastcomp-clang and then wants to replace each of these with one's own fork, then yes, one would go into the build directory (e.g. `C:\emsdk\clang\fastcomp\build_incoming_vs2017_64`), and issue `cmake ../src` (to target `C:\emsdk\clang\fastcomp\src` which is where the sources are cloned to)

But generally if one wants to just use one's own fork of Emscripten, they'd do

```bash
git clone https://github.com/juj/emsdk.git
cd emsdk
emsdk install sdk-incoming-64bit
emsdk activate sdk-incoming-64bit
source ./emsdk_env.sh
cd emscripten/incoming
git remote add my_own_fork https://github.com/my_own_fork/emscripten.git
git fetch my_own_fork
git checkout my_branch
```

and they'd be all set, i.e. this scheme reuses kripken/emscripten-fastcomp and kripken/emscripten-fastcomp-clang, but moves the working tree in emscripten/incoming over to the custom fork.

Comment (gabrielcuvillier) [reply site/source/docs/building_from_source/building_emscripten_from_source_using_the_sdk.rst]: Yes, I found weird to have to fork fastcomp and fastcomp-clang also, even if I only want to work on emscripten...

**Comment by gabrielcuvillier** [comment]
This is no more relevant now. I close this for cleaning up PRs

**Pull request closed without merging**